guides/plugins/apps/app-base-guide.md

```diff
@@ -116,12 +121,12 @@ You and the Shopware Account are the only parties that should know your `app-sec
 {% endhint %}
 
 {% hint style="warning" %}
-For local development you can specify a &lt;secret&gt; in the manifest file, that is used for signing the registration request. However if a app uses a hard-coded secret in the manifest it can not be uploaded to the store.
+For local development you can specify a &lt;secret&gt; in the manifest file, that is used for signing the registration request. However, if an app uses a hard-coded secret in the manifest it can not be uploaded to the store.
 {% endhint %}
 
 To verify that the registration can only be triggered by authenticated Shopware shops you need to recalculate the signature and check that the signatures match, thus you've verified that the sender of the request possesses the `app secret`.
 
-Following code snippet can be used to recalculate the signature:
+Thw following code snippet can be used to recalculate the signature:
```

Comment:
```suggestion
The following code snippet can be used to recalculate the signature:
```

Or do you really meant the `Technisches Hilfswerk` :trollface:

Reply:
🙈